app/config/config_dev.yml

```diff
@@ -14,15 +14,18 @@ web_profiler:
 monolog:
     handlers:
         main:
-            type:  stream
-            path:  %kernel.logs_dir%/%kernel.environment%.log
-            level: debug
+            type:   stream
+            path:   %kernel.logs_dir%/%kernel.environment%.log
+            level:  debug
+        console:
+            type:   console
+            bubble: false
```

Comment:
@Tobion Yes I know and my "issue" was not about firephp. It was about DX ; to remove a possible WTF effect.